pyproject.toml

```diff
@@ -25,6 +25,7 @@ python = "^3.8"
 pymongo = {extras = ["srv"], version = "^3.12.1"}
 requests = {extras = ["crypto"], version = "^2.26.0"}
 celery = {extras = ["tblib"], version = "^5.2.0"}
+pydantic = "^1.9.0"
```

Comment:
You probably don't need 

```
isort = "^5.10.1"
mypy = "^0.931"
black = "^21.10b0"
pyupgrade = "^2.29.1"
flake8 = "^4.0.1"
pre-commit = "^2.15.0"
flake8-black = "^0.2.3"
```

from your dev dependencies.

Reply:
Right, deleted